lma-ai-stack/source/lambda_layers/transcript_enrichment_layer/eventprocessor_utils/eventprocessor.py

```diff
@@ -49,11 +56,39 @@ def get_meeting_ttl():
 def get_transcription_ttl():
     return get_ttl(TRANSCRIPTION_RECORD_EXPIRATION_IN_DAYS)
 
-def get_owner_from_jwt(jwt_token):
-    decoded_jwt = jwt.decode(jwt_token, options={"verify_signature": False})
-    print("DECODED JWT", decoded_jwt)
-    return decoded_jwt['username']
+def get_owner_from_jwt(jwt_token, verifySignature):
```

Comment:
I don't recall why we added this to the event processor, in the context of UBAC sharing.. 
Are you sure we need to verify the JWT at this point.. since it was already verified by websocket server..  Can the JWT expire during the call and cause problems?